src/i18n.js

```diff
@@ -0,0 +1,15 @@
+import { createI18n } from "vue-i18n";
+import enTrans from "./locale/en.json";
```

Comment:
```suggestion
import englishTranslations from "./locale/en.json";
```
There is absolutely no need to take shortcut in names at expense of clarity.